en/concept/flight_modes.md

```diff
@@ -41,7 +41,7 @@ The sections below provide an overview of the modes, followed by a [flight mode
   * **Multirotors:**  The multirotor hovers / loiters at the current position and altitude.
 * **AUTO_RTL** (Return to Land)
   * **Fixed wing aircraft:** The aircraft returns to the home position and loiters in a circle above the home position. 
-  * **Multirotors:** The multirotor returns in a straight line on the current altitude (if higher than the home position + loiter altitude) or on the loiter altitude (if higher than the current altitude), then lands automatically.
+  * **Multirotors:** The multirotor returns in a straight line on the current altitude (if the current altitude is higher than the home position + loiter altitude) or on the loiter altitude (if the loiter altitude is higher than the current altitude), then lands automatically.
```

Comment:
Actual behaviour is a bit more complicated than this, and is affected by parameters (e.g. a plane might land). The "loiter altitude" is not the correct parameter - should be RTL return height. We should tidy this a little and link to https://docs.px4.io/en/flight_modes/rtl.html.  Actually we should link to all the dedicated pages in user guide for each of these.

Reply:
Agreed. Do I have to install Gitbook editor to insert the link?

Reply:
No, you can add links manually. An external link in markdown has the format 
`[Text to display](Link URL)`.